src/planeta/HandshakeButtons.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { Text, Heading, QR as QRCode } from "rimble-ui";
+import { BorderButton } from "../components/Buttons";
+import { choice } from "./utils";
+
+const COLLABORATE_EMOJIS = ["🥰", "🌹", "🍻", "🥂"];
```

Comment:
Missing emoji here? :)

Reply:
Feel free to suggest more emojis :dancer: